sdks/python/apache_beam/typehints/native_type_compatibility.py

```diff
@@ -36,37 +36,38 @@
     '_TypeMapEntry', ['match', 'arity', 'beam_type'])
 
 
+def _get_compatible_args(typ):
+  # On Python versions < 3.5.3, the Tuple and Union type from typing do
+  # not have an __args__ attribute, but a __tuple_params__, and a
+  # __union_params__ and __union_set_params__ argument respectively.
```

Comment:
nit: we can remove mention of  `__union_set_params__ ` - it is not relevant in this code path and might distract the reader. It stores the same `__union_params__` in the set.